concrete/src/Package/Package.php

```diff
@@ -631,28 +641,22 @@ public function getPackageEntityPaths()
     public function installDatabase()
     {
         $this->installEntitiesDatabase();
-
         static::installDB($this->getPackagePath() . '/' . FILENAME_PACKAGE_DB);
     }
 
     public function installEntitiesDatabase()
```

Comment:
This isn't anything new but it'd be nice to have a PackageInstaller class that managed this instead of having it as part of the package itself because having the interface for Package is already pretty dang big